.github/workflows/snyk-pr.yml

```diff
@@ -0,0 +1,31 @@
+name: snyk-pr
+on:
+  pull_request:
+    branches:
+      - master
+jobs:
+  whitesource:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.pull_request.user.login == 'sfc-gh-snyk-sca-sa' }}
+    steps:
+    - name: checkout
+      uses: actions/checkout@v3
+      with:
+        ref: ${{ github.event.pull_request.head.ref }}
+        fetch-depth: 0
+
+    - name: checkout action
+      uses: actions/checkout@v3
+      with:
+        repository: snowflakedb/whitesource-actions
+        token: ${{ secrets.WHITESOURCE_ACTION_TOKEN }}
```

Comment:
Again why are we using WHITESOURCE this often. This can cause confusion.

Reply:
Hey, @sfc-gh-hchaturvedi  the WHITESOURCE_ACTION_TOKEN means the github token to the script we use for automation. The repo name is called `whitesource-action` => https://github.com/snowflakedb/whitesource-actions

it actually has nothing to do with real whitesource app